### ISSUE TITLE
Email Verification doesn't work

### ISSUE THREAD

**Issue by Marios1231** [hrfee/jfa-go#150]
When I turn on the option and test I get a link and when I press it shows 
Page not found.
Need help? contact me.
![Screenshot 2021-09-15 160225](https://user-images.githubusercontent.com/52550609/133379211-7dd92789-ffb6-42cb-95a4-552cb4ad4cef.png)

**Comment by hrfee** [comment]
What's the full URL that you get? This might be related to the url base setting.

**Comment by SteveGBuck** [comment]
I'm getting the same.

The link in the email is (replaced my real domain with example.com):

https://jellyfin.example.com/accounts/invite/hbokBeCs75UaXrxdHXX6BU?key=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJlbWFpbCI6InN0ZXZlZ2J1Y2tAb3V0bG9vay5jb20iLCJleHAiOiIxNjMxOTM0MjI0IiwiaW52aXRlIjoiaGJva0JlQ3M3NVVhWHJ4ZEhYWDZCVSIsInBhc3N3b3JkIjoiTG9vc2VuNTUyNCQiLCJ0ZWxlZ3JhbVBJTiI6IiIsInR5cGUiOiJjb25maXJtYXRpb24iLCJ1c2VybmFtZSI6InN0ZXZlZ2J1Y2tAb3V0bG9vay5jb20iLCJ2YWxpZCI6dHJ1ZX0.EVLKw6VLTMHqH0ams4LFwlrg8yxesjM-5bw2W8XSnPA

**Comment by Marios1231** [comment]
https://www.example.com/accounts/pFFyFAbtyKAqL2bPPJKdyD?key=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJlbWFpbCI6Im1haHJpZjM1MEBnbWFpbC5jb20iLCJleHAiOiIxNjMxOTM1NTEwIiwiaW52aXRlIjoicEZGeUZBYnR5S0FxTDJiUFBKS2R5RCIsInBhc3N3b3JkIjoiTWFyaW9zOTk4IiwidGVsZWdyYW1QSU4iOiIiLCJ0eXBlIjoiY29uZmlybWF0aW9uIiwidXNlcm5hbWUiOiJNYXJ5dXMiLCJ2YWxpZCI6dHJ1ZX0.KeyIZG2VNd7vL2igG5FBQJLXM3dTj6bA8K47H-48dvM

**Comment by hrfee** [comment]
@SteveGBuck The URL includes `/accounts` which is correct, but shows you a 404? Does opening a normal invite at `jellyfin.example.com/accounts/invite/<invite code>` work? This all seems to be working fine for me with URL base set to `/accounts` in Settings > General, but I haven't yet tested it with a reverse proxy.

**Comment by SteveGBuck** [comment]
Yep, invite works perfectly. (I've set "/accounts" in URL base).  Just the email verification not working.

**Comment by Marios1231** [comment]
invite works for me too just no the email verification 

Harvey maybe you can set up jfa-go and see if it works for you

**Comment by recom273** [comment]
Im getting the same error

http://jfa.krylon.live/accounts/VDavTZjiC9qNDJyeBcw9UW?key=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJlbWFpbCI6InJlY29tMjczQGdtYWlsLmNvbSIsImV4cCI6IjE2MzIxODIyNDEiLCJpbnZpdGUiOiJWRGF2VFpqaUM5cU5ESnllQmN3OVVXIiwicGFzc3dvcmQiOiJCeWtubzUtbnVtZ3VrLXR1dmt1YyIsInRlbGVncmFtUElOIjoiIiwidHlwZSI6ImNvbmZpcm1hdGlvbiIsInVzZXJuYW1lIjoicmVjbyIsInZhbGlkIjp0cnVlfQ.OR2RDCUXDIPiADhtW_dMfzxn1GE8DVMw_0ynmfFidL0

`[GIN] GET(/invite/VDavTZjiC9qNDJyeBcw9UW?key=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJlbWFpbCI6InJlY29tMjczQGdtYWlsLmNvbSIsImV4cCI6IjE2MzIxODIyNDEiLCJpbnZpdGUiOiJWRGF2VFpqaUM5cU5ESnllQmN3OVVXIiwicGFzc3dvcmQiOiJCeWtubzUtbnVtZ3VrLXR1dmt1YyIsInRlbGVncmFtUElOIjoiIiwidHlwZSI6ImNvbmZpcm1hdGlvbiIsInVzZXJuYW1lIjoicmVjbyIsInZhbGlkIjp0cnVlfQ.OR2RDCUXDIPiADhtW_dMfzxn1GE8DVMw_0ynmfFidL0) => 404
[ERROR] 18:57:36 views.go:248: Failed to parse key: Token is expired

[GIN] GET(/invite/VDavTZjiC9qNDJyeBcw9UW?key=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJlbWFpbCI6InJlY29tMjczQGdtYWlsLmNvbSIsImV4cCI6IjE2MzIxODIyNDEiLCJpbnZpdGUiOiJWRGF2VFpqaUM5cU5ESnllQmN3OVVXIiwicGFzc3dvcmQiOiJCeWtubzUtbnVtZ3VrLXR1dmt1YyIsInRlbGVncmFtUElOIjoiIiwidHlwZSI6ImNvbmZpcm1hdGlvbiIsInVzZXJuYW1lIjoicmVjbyIsInZhbGlkIjp0cnVlfQ.OR2RDCUXDIPiADhtW_dMfzxn1GE8DVMw_0ynmfFidL0) => 404
[ERROR] 18:57:36 views.go:248: Failed to parse key: Token is expired
`

I had this working successfully behind a reverse proxy for over 6 months already, what happened?

**Comment by BobDaGecko** [comment]
Same issue here.

```
Oct 01 05:23:24 server jfa-go[1644983]: [GIN] GET(/SpjCNxvCD8JQgMczNhgMV3?key=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJlbWFpbCI6ImJvYmRhZ2Vja29idXNpbmVzc0BnbWFpbC5jb20iLCJleHAiOiIxNjMzMTA4OTE2IiwiaW52aXRlIjoiU3BqQ054dkN>
```

**Comment by hrfee** [comment]
It looks like another issue with the new JWT (auth token) library, the previous one is no longer maintained so I had to switch to a new one. Sorry for the delay on this also, just started Uni so i've been a bit busy.

**Comment by hrfee** [comment]
@BobDaGecko your issue looks a little different, there's no `/invite` in the URL. Which version are you running, and what do you have set in Settings > Invite Emails > URL Base?

**Comment by hrfee** [comment]
@Marios1231 this all works for me. The URL you sent doesn't include `/invite` either, could you try again but before visiting the URL, add `invite/` after `/accounts`? e.g `https://www.example.com/accounts/invite/pFFyFAbtyKAqL2bPPJKdyD?key=eyJ......`

**Comment by BobDaGecko** [comment]
I made sure the Invite Email was example.com/invite. It is what I had when I first posted. Despite this, I still got the same error.

**Comment by Marios1231** [comment]
> @Marios1231 this all works for me. The URL you sent doesn't include `/invite` either, could you try again but before visiting the URL, add `invite/` after `/accounts`? e.g `https://www.example.com/accounts/invite/pFFyFAbtyKAqL2bPPJKdyD?key=eyJ......`


Hey man i just tried it and it still didn't work but it's fine if you've got uni to focus on, i can't speak for all of us but take your time I will wait.

**Comment by Marios1231** [comment]
if it helps my invite url base is also accounts
